data/common.yaml

```diff
@@ -11,3 +10,21 @@ grafana::config_file: "%{lookup('grafana::config_dir')}/grafana.ini"
 
 grafana::user: grafana
 grafana::group: grafana
+grafana::settings::auth_ldap::ldap_servers:
+  - localhost
+grafana::settings::auth_ldap::enabled: true
```

Comment:
Sorry for not seeing this before but I think LDAP authentication should be disabled by default.

Reply:
My thinking hear is that by default ldap wont be enabled.  it is only enabled if someone calls

```puppet
class grafana {
  settings => {
    'auth.ldap' => { ...}
  }
}
```
or `class { 'grafana::settings::auth_ldap': }` directly if one considers this then i think a default of enabled makes more senses but happy to change.

Reply:
You're right, once again.